react/src/component/login/email.tsx

```diff
@@ -22,20 +22,21 @@ export const LoginEmailConfirm = (props) => {
             headers: {
                 "Content-Type": "application/json"
             }
-        }).then(() => 
+        }).then(() =>
         updateLoginStatusRef(loginStatusRef, appWrapper)
-        .then(() => navigate(`/react/${appName}`))
+        .then(() => navigate(`/${appName}`))
         )
     }
+
     return (
         <Container>
             {
-                needCompleteRegistration ? (
+                needCompleteRegistration === '1' ? (
                     <LoginExtraInfoForm onComplete={(data) => completeLogin(data)}/>
                 ) : (
                     <>
-                        <p>Click the button complete the login process.</p>
-                        <Button onClick={() => completeLogin()}>Continue</Button>
+                        <p>Please click the button below complete the login process.</p>
```

Comment:
I have asked numerous times for us to use string tables. This embedded stuff is going to get harder and harder to maintain. The style of "Please click" vs "Click" and "Continue" vs "Confirm" should be auditable, easily, in a string file.